admin/src/pages/List.jsx

```
@@ -0,0 +1,84 @@
+              alt=""
+            />
+            <p>{item.name}</p>
+            <p>{item.category}</p>
+            <p>
+              {currency}
+              {item.price}
+            </p>
+            <p
+              onClick={() => removeProduct(item._id)}
+              className="text-center text-white w-[25%] bg-gray-700 md:text-center md:w-full cursor-pointer hover:bg-red-700 md:text-white px-4 py-2 sm:px-7 sm:py-2 rounded-full text-xs sm:text-sm"
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)